internal/impl/devbox.go

```diff
@@ -455,7 +455,7 @@ func (d *Devbox) PrintEnv() (string, error) {
 		}
 		return script, nil
 	}
-	envs, err := d.computeNixEnv(false)
```

Comment:
huh.. this is weird and maybe a bug.. I think this should be `true`. This is what `direnv` integration uses, so we definitely want to set the full path here.

Or am I missing something?

Reply:
`false` assumes the path gets built in the rc files but that only happens for `devbox shell` right? for direnv, I assume it should be true, but not confident. I'll fix it in new PR and test properly